css/burgermenu.css

```diff
@@ -0,0 +1,49 @@
+.burger-menu__container {
+  display: block;
+  position: fixed;
+  height: 100vh;
+  width: 300px;
+  background: var(--color-dark);
+  z-index: 10;
+  left: -100%;
+  transition: left 0.8s ease-in-out;
+}
+
+.open {
+  left: 0;
+}
+
+.burger-menu__content {
+  display: flex;
+  flex-direction: column;
+}
+
+.burger-menu__banner {
+  width: 300px;
+  height: 200px;
+}
+
+.burger-menu_nav > .nav-list {
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+  width: fit-content;
+}
+
+.burger-menu__open-btn {
```

Comment:
A minor thing, I believe an open/close is rather a modifier, so naming can be like `burger-menu__btn--open` etc...